packages/core/addon/templates/components/visualization-config/chart-type/date-time.hbs

```diff
@@ -1,5 +1,2 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for
-terms. --}}
-<div class="visualization-config__warn-msg">
-  No configuration options available.
-</div>
\ No newline at end of file
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for
+terms. --}}
```

Comment:
What is this change for?

Reply:
The message "no configuration" is no longer relevant as there is configuration.  eventually we'll refactor this out. but this is a temporary punt to keep things focused.